### PR TITLE
cv sergio costa

### PR DESCRIPTION
**Pull request by Scostta**
Sergio Costa CV

### CODE REVIEW

File: staff/sergio-costa/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <title>CV Sergio Costa</title>
+</head>
+<body>
+    <header>
+        <h1>CV Sergio Costa</h1>
+    </header>
+    <section>
+        <h3>Datos personales</h3>
+        <ul>
+            <li><b>Direccion:</b> C/Josep Irla 27</li>
```
Comment (lexbarba) [inline staff/sergio-costa/cv/index.html]: The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the to
tags, emphasized text should be denoted with the tag, important text should be denoted with the tag, and marked/highlighted text should use the tag. https://www.w3schools.com/tags/tag_b.asp

Comment (robert-z) [inline staff/sergio-costa/cv/index.html]: The W3C [discourages](https://developer.mozilla.org/es/docs/Web/HTML/Elemento/b#Notas) the use of `b` element.

Comment (Scostta) [reply staff/sergio-costa/cv/index.html]: fixed 👍

Comment (Scostta) [reply staff/sergio-costa/cv/index.html]: fixed ! 👍

**Comment by nicopixel** [comment]
🖕

File: staff/sergio-costa/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+<body>
+    <header>
+        <h1>CV Sergio Costa</h1>
+    </header>
+    <section>
+        <h3>Datos personales</h3>
+        <ul>
+            <li><strong>Direccion:</strong> C/Josep Irla 27</li>
+            <li><strong>Movil:</strong> 676690541</li>
+            <li><strong>Email:</strong> sergio.cs04@gmail.com</li>
```
Comment (quinwacca) [inline staff/sergio-costa/cv/index.html]: You can use `<a href="mailto: ....>` and `<a href:="tel: ....>` to make others contact you easier.

**Pull request merged**